spyder/plugins/layout/plugin.py

```diff
@@ -753,6 +753,7 @@ def create_plugins_menu(self):
         Populate panes menu with the toggle view action of each base plugin.
         """
         order = ['editor', 'ipython_console', 'variable_explorer',
+                 'frames_explorer',

```

Comment:
Please move the other elements below on this list here, so that this line does not only contain `'frames_explorer'`.